pubsub/api/Pubsub.Samples/PullMessagesAsyncWithDeliveryAttempts.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START pubsub_dead_letter_delivery_attempt]
```

Comment:
Number of delivery attempts is best effort only. What this field will print may be different from what the service sees. So we want to avoid showing an example as suggested.